scenes/cslib_array.txt

```diff
@@ -155,7 +155,7 @@
 *comment        p_value(any): the value to be added to the end of the array
 *label push
 *params p_array_ref p_value
-*gosub insert p_array_ref p_value ({{p_array_ref&"_count"}} + 1)
```

Comment:
maybe we could make it uniform across functions, so that the prefix includes the underscore (for stuff like {prefix & x} )

Reply:
That'd be putting the onus on the user. Yes, it makes our code the tiniest bit easier to read, but we're trying to accommodate end-users who don't have a programming background here -- I think that would be a move in the wrong direction.